src/containers.js

```diff
@@ -176,7 +178,7 @@ exports.decorateTerm = (Term, { React }) => {
       term.selectionManager._model.selectionStart = [_startIdx, _startRow];
       term.selectionManager._model.selectionEnd = [_endIdx + 1, _endRow];
       term.selectionManager.refresh();
-      term.scrollDisp(_startRow - term.buffer.ydisp);
+      term.scrollLines(_startRow - term.buffer.ydisp);
```

Comment:
I noticed there was no `term.scrollDisp` command?  I'm not sure if they changed it or not.  It was only scrolling back and forth between the first and last results.

Changing this to `scrollLines` made things work.